src/item/enchantment/EnchantmentHelper.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\item\enchantment;
+
+use pocketmine\block\BlockTypeIds;
+use pocketmine\item\Armor;
+use pocketmine\item\Book;
+use pocketmine\item\Item;
+use pocketmine\item\TieredTool;
+use pocketmine\utils\Random;
+use pocketmine\world\Position;
+use function abs;
+use function array_filter;
+use function count;
+use function max;
+use function min;
+use function range;
+
+final class EnchantmentHelper{
+	public const MAX_BOOKSHELF_COUNT = 15;
+	public const FIRST_OPTION = 0;
+	public const SECOND_OPTION = 1;
+	public const THIRD_OPTION = 2;
```

Comment:
I can remove the `FIRST_OPTION`, `SECOND_OPTION`, `THIRD_OPTION` constants, but I don't think it's a good idea to remove `MAX_BOOKSHELF_COUNT` as it will reduce the readability of the code